content/docs/hooks-intro.md

```diff
@@ -25,86 +25,90 @@ function Example() {
 }
 ```
 
-This new function `useState` is the first "Hook" we'll learn about, but this example is just a teaser. Don't worry if it doesn't make sense yet!
+הפונקציה החדשה הזאת `useState` היא ה"Hook" הראשון שנלמד. דוגמא זו היא רק הצצה, אל תדאג אם זה עדיין לא נשמע הגיוני!
 
-**You can start learning Hooks [on the next page](/docs/hooks-overview.html).** On this page, we'll continue by explaining why we're adding Hooks to React and how they can help you write great applications.
+**אתה יכול להתחיל ללמוד Hooks [בעמוד הבא](/docs/hooks-overview.html).** בעמוד זה, נמשיך להסביר למה הוספנו את Hooks ל-React ואיך הם יכולים לעזור לך לכתוב יישומים מעולים.
 
->Note
+>הערה
 >
->React 16.8.0 is the first release to support Hooks. When upgrading, don't forget to update all packages, including React DOM. React Native will support Hooks in the next stable release.
+>React 16.8.0 היא הגרסא הראשונה שתומכת ב-Hooks. בעת שדרוג, לא לשכוח לעדכן את כל הספריות, כולל React Native .React Dom יתמוך ב-Hooks בגרסא היציבה הבאה.
 
-## Video Introduction {#video-introduction}
 
```

Comment:
Try to keep same line numbers to make it easier to merge changes later in the future, please remove empty line